core/modules/file/file.module

```diff
@@ -2613,16 +2622,6 @@ function _file_sort_array_by_weight(&$a) {
   }
 }
 
-/*
- * Implements hook_cron_queue_info().
- */
-function file_cron_queue_info() {
-  $queues['file_type_determine'] = array(
-    'worker callback' => 'file_type_determine',
-  );
-  return $queues;
-}
```

Comment:
I think removing this queue is a good move, the batch processing page is a lot more accessible. But we might want to have another update hook to remove this queue if it exists? Otherwise either an error might occur trying to process the queue on cron jobs, or maybe the queue will just never be emptied?

Reply:
Added a new update hook implementation to remove the queue.